Src/IronPython.Modules/_codecs.cs

```diff
@@ -182,41 +184,37 @@ public static PythonTuple escape_decode(CodeContext/*!*/ context, [NotNull]strin
         [DisallowNull]
         private static byte[]? _replacementMarker;
 
-        public static PythonTuple/*!*/ escape_encode([BytesLike,NotNull]ReadOnlyMemory<byte> data, string? errors = null) {
-            var buf = new List<byte>(data.Length);
-            var span = data.Span;
+        public static PythonTuple/*!*/ escape_encode([NotNull]IBufferProtocol data, string? errors = null) {
```

Comment:
While testing, I noticed this method only accepts `Bytes` in CPython.

Reply:
Interesting, it would be good to check all of them by adding more tests.